src/app/pages/page-two/page-two.component.css

```diff
@@ -0,0 +1,23 @@
+.card-section {
+  margin: 32px;
+  display: grid;
+  gap: 16px;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
+.card-section tds-card {
+  display: grid;
```

Comment:
Should the card really have display grid?

Reply:
No, you are right. I dropped it and simplified it.